src/modules/theme/components/header/header-archived-banner.component.ts

```diff
@@ -27,15 +27,21 @@ export class HeaderArchivedBannerComponent implements OnInit {
     this.isAdmin = this.authentication.isAdminRole();
     this.innovation = this.context.getInnovation();
     this.isOwner = this.innovation.loggedUser.isOwner;
-    this.baseUrl = `${this.authentication.userUrlBasePath()}/innovations/${this.innovation.id}`;
-    this.regEx = new RegExp(`(${this.baseUrl.replace(/\//g, '\\/')}\/)(manage\/innovation?|[a-zA-Z-]*)$`);
+
+    this.regEx = new RegExp(/innovations\/[\w\-]+\/([\w\-]+|manage\/innovation)(\?.*)?$/);
+    this.showBanner = this.innovation.status === 'ARCHIVED' && this.regEx.test(this.router.url);
```

Comment:
You can use the `this.checkShowBanner()` function instead of replicating the logic